packages/utils/src/ts-eslint/Linter.ts

```diff
@@ -316,7 +316,7 @@ namespace Linter {
 
   export interface ESLintParseResult {
     ast: TSESTree.Program;
-    parserServices?: ParserServices;
+    services?: ParserServices;
```

Comment:
i don't think that anyone used this type before, but this was not correct, it has to be `services`